docs/FAQ.md

```diff
@@ -57,6 +57,10 @@ More information about cards that support forward compatibility can be found
 
 ### How can I check if the RAPIDS Accelerator is installed and which version is running?
 
+Using the `--jars` or `--packages` option followed by the file path or maven path to RAPIDS jar is the preferred way to run RAPIDS accelerator.
+If RAPIDS jar is copied directly to the `$SPARK_HOME/jars` folder, it might result in the ClassLoader not setting up classes
```

Comment:
Having rapids jars as part of extraClassPath triggers the same Exception as copying to `$SPARK_HOME/jars`. 
I would not even mention latter because it seems like a user error. 

Not sure what we want to call extraClassPath  officially. I used the to use the term static classpath because they are prepended independently of job classes (--jars, --packages).  @tgravescs might want to chime in